### DISCUSSION TITLE
Will librelink on my phone stop if i use xdrip on my garmin?

### DISCUSSION THREAD

**Discussion by Jennyninja** [NightscoutFoundation/xDrip#2704] [General]
Hi, I've just found this group. Before I get started I wanted to check will scan results on my phone stop if I use xdripfor my Garmin?

**Comment by Navid200** [comment]
@psonnera

**Reply by PhilTU** [reply]
Scanning (NFC) is a manual action, you can still do that with Librelink while xDrip is linked to the sensor by Bluetooth. Easier if you open Librelink first.

**Comment by Jennyninja** [comment]
Thanks for clarification!

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
